Shared/en.lproj/Localizable.strings

```diff
@@ -35,7 +35,7 @@
 "Custom title" = "Custom title";
 
 /* No comment provided by engineer. */
-"Don't send Content Security Policy header (allows your website to use third-party resources)" = "Don't send Content Security Policy header (allows your website to use third-party resources)";
+"Don't send Content Security Policy header (allows your website to use third-party resources)" = "Don't send header allowing your website to use third-party resources";
```

Comment:
In similar vein to https://hosted.weblate.org/translate/onionshare/translations/nb_NO/?checksum=7b9283818fa84a0a

Reply:
Your suggestion 

> Don't send header allowing your website to use third-party resources

could also be read  as "don't send that header which allows third-party resources". 

Which is the opposite of what it does.

Let's turn this upside-down (or rather into the user's perspective):

How about:

> Allow third-party resources. (Disables the safe default Content-Security-Policy header.)

This explains what it does from the user perspective and also stays technically correct to what actually happens for the folks who want to understand what's going on behind the scenes.

Do you think this reads right?

Reply:
Good catch. Meant to put a comma in there.
I like the active and switched case. It does fall apart for me in the second sentence though.
("Disable" is only used in two options and one description throughout OnionShare.)

If explaining that it is ones website can be omitted (what happens), then why not also the technical details (how it is done)?
It is explained (badly) in https://hosted.weblate.org/translate/onionshare/doc-features/en/?checksum=10280854161c2e36

```suggestion
"Permit loading third-party resources";
```
Gives it a nuance of risk trade-off.

Reply:
Just for technicalities: This change brakes the `strings` file format, because now the key is missing.

I'm a little wary of reducing it down to this for the following reasons:

1. The `loading` (or not loading) takes place in the web browser of the other party of the hosting.
2. It's totally up to that other browser, if it honours the `Content-Security-Policy` header.
3. This is in the "hosting" mode, where the phone just becomes a web server. You need to understand some technical stuff, if you want to use that. You only want to check this box, if you e.g. linked to some script on a content delivery network in your *self-written* html file. The CSP header we normally send, disables that for the typically used web browsers.

Reply:
Ok, I will change it to "Permit third-party resources (don't send Content Security Policy header)".

Because not sending a CSP header has other implications (which there is not enough space to explain). And some folks who are using this feature are actually capable of understanding these.